spec/Factory/CsvSerializerFactorySpec.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
Missing OS Header. Please see all occurrences like this, as it's not the single case.